calour/util.py

```diff
@@ -43,25 +44,100 @@
 logger = getLogger(__name__)
 
 
+def join_fields(df, field1, field2, joined_field=None, sep='_', pad=None):
```

Comment:
I think this should be in experiment rather than calour in general... you do this function on an experiment with dataframes in it...